ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/integration/PullQueryRoutingFunctionalTest.java

```diff
@@ -292,6 +292,63 @@ public void cleanUp() {
     APP_SHUTOFFS_2.reset();
   }
 
+  @Test
+  public void shouldReturnLimitRowsMultiHostSetup() {
```

Comment:
You should probably do this in its own file since these tests are very specific to routing.

Also, this has only one partition in these tests, which honestly is not a very interesting mult-node case.  Would be great to have a tests with a couple nodes and at least a couple partitions.